### PR TITLE
[Docs] translate doc/c_api_quick_start to simplified Chinese

### PR DESCRIPTION
**Pull request by kenvifire**
translate doc/c_api_quick_start to simplified Chinese

### CODE REVIEW

**Comment by hydai** [comment]
Hi @kenvifire,
中英文間應都以空格隔開會比較合適，如 `中文English中文` 應該改為 `中文 English 中文`

**Comment by kenvifire** [comment]
@hydai 
Spaces added, please check again, thanks!

**Comment by kenvifire** [comment]
Hi @hydai 
Any update on this MR?

File: docs/c_api_quick_start_zh.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# WasmEdge C API 文档
+
+文档 [WasmEdge C API](../include/api/wasmedge.h.in) 里提供了访问 WasmEdge 运行时的接口。以下内容是关于如何使用 WasmEdge C API 的简单说明。如果要了解 WasmEdge C API 的更多细节，请参考[完整的文档](c_api.md)。
+
+## WasmEdge 运行时快速指南
+
+以下是运行一个 WASM 文件的示例。
+假设 WASM 文件[`fibonacci.wasm`](../tools/wasmedge/examples/fibonacci.wasm)已经被拷贝到当前目录，并且C文件`test_wasmedge.c`的源代码如下所示：
```
Comment (hydai) [inline docs/c_api_quick_start_zh.md:8]: `并且C文件`test_wasmedge.c`的源代码如下所示`
->
`并且 C 文件 `test_wasmedge.c` 的源代码如下所示`

File: docs/c_api_quick_start_zh.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+  /* 资源析构。 */
+  WasmEdge_VMDelete(VMCxt);
+  WasmEdge_ConfigureDelete(ConfCxt);
+  WasmEdge_StringDelete(FuncName);
+  return 0;
+}
+```
+
+接下来你可以编译并且运行它：（索引从0开始的情况下，斐波那契数列的的第32个数值是5702887）
```
Comment (hydai) [inline docs/c_api_quick_start_zh.md:43]: The number should also have a space between Chinese characters.

File: docs/c_api_quick_start_zh.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+$ gcc test_wasmedge.c -lwasmedge_c -o test_wasmedge
+$ ./test_wasmedge fibonacci.wasm
+Get result: 5702887
+```
+
+如果要了解 API 的细节，请参考 [API 头文件](../include/api/wasmedge.h.in).
+
+## WasmEdge AOT 编译器快速指南
+
+假设 WASM 文件 [`fibonacci.wasm`](../tools/wasmedge/examples/fibonacci.wasm) 已经被拷贝到当前目录，并且C文件`test_wasmedge_compiler.c`的代码如下所示：
```
Comment (hydai) [inline docs/c_api_quick_start_zh.md:55]: Ditto.

File: docs/c_api_quick_start_zh.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      return 1;
+  }
+
+  WasmEdge_CompilerDelete(CompilerCxt);
+  WasmEdge_ConfigureDelete(ConfCxt);
+  return 0;
+}
+```
+
+接下来你可以编译和运行它（输出的文件是"fibonacci.wasm.so"）：
```
Comment (hydai) [inline docs/c_api_quick_start_zh.md:82]: Ditto.

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by hydai** [comment]
Please do not use `create a merge commit` to sync with the latest master, this will make the DCO check fail.
To fix this, try to use `fast-forward merge` or `rebase` to correct the commit history. Thanks.

**Review by hydai — APPROVED** [review]

**Pull request merged**